inst/validation/bbr-requirements.yaml

```diff
@@ -566,6 +566,10 @@ NMJ-R006:
   description: nm_join() warns on skipping table with wrong number of rows
   tests:
   - BBR-NMJ-006
+NMJ-R007:
+   description: nm_join() informatively errors if .join_col has duplicate values in any of the input tables
+   tests:
+  - BBR-NMJ-007
```

Comment:
Can you add `NMJ-R007` to a story? Think it makes sense to add it to `OUT-S003`:

```yaml
OUT-S003:
  name: Join NONMEM tables to input data
  description: As a user, I want to be able to pass a model object and have bbr extract
    the input data path and path to any table outputs from the control stream and
    join the contents of those files together into a single tibble.
  ProductRisk: Low
  requirements: 
   - NMJ-R001
   - NMJ-R007
```